cocotb/regression.py

```diff
@@ -472,19 +473,26 @@ def execute(self) -> None:
                 return self._start_test()
 
     def _start_test(self) -> None:
+        # Want this to stand out a little bit
         start = ''
         end = ''
         if want_color_output():
             start = ANSI.COLOR_TEST
             end = ANSI.COLOR_DEFAULT
-        # Want this to stand out a little bit
+
+        if self._test.__doc__ is not None:
+            description = _trim(self._test.__doc__)
+        else:
+            description = ""
+
         self.log.info(
-            "{start}running{end} {name} ({i}/{total})".format(
+            "{start}running{end} {name} ({i}/{total}){description}".format(
                 start=start,
                 i=self.count,
                 total=self.ntests,
                 end=end,
                 name=self._test.__qualname__,
+                description=description,
```

Comment:
`_trim` already handles empty docstrings.
```suggestion
                description=_trim(self._test.__doc__),
```